### ISSUE TITLE
NTR: apoptotic process in response to mitochondrial fragmentation (was: regulation of mitochondrial fragmentation involved in apoptotic process)

### ISSUE THREAD

**Issue by slaulederkind** [geneontology/go-ontology#15045]
I would like to see "regulation of mitochondrial fragmentation involved in apoptotic process”, 
positive regulation of…” and " negative regulation of…” added to the GO ontology under the parent term "mitochondrial fragmentation involved in apoptotic process (GO:0043653).  The definitions could be "Any process that modulates/increases/decreases the frequency, rate or extent of the change in the morphology of the mitochondria in an apoptotic cell from a highly branched network to a fragmented vesicular form.”  Example can be found in PMID:18940801.

**Comment by pgaudet** [comment]
hi @slaulederkind 

Which protein/figure are you annotating? I believe this is a readout for apoptosis. Could you not annotate to just 'regulation of apoptotic process'? 

Thanks, Pascale

**Comment by slaulederkind** [comment]
Figure 5 is connecting mitochondrial fission to apoptosis.  Blocking Fis1 decreases fission and decreases apoptosis.

**Comment by pgaudet** [comment]
Thanks. From the paper I would rather create 'apoptosis in response to mitochondrial fragmentation': (rather than regulating it): 
'mitochondrial fragmentation occurs upstream of apoptosis and is both necessary and sufficient for hippocampal neuron death'; see also diagram in Figure 6 C

My understanding is that a change in balance of mitochondrion fusion/fission triggers Bcl2 to cause apoptosis. 

What do you think ? 

Thanks, Pascale

**Comment by slaulederkind** [comment]
I was going by this rule of thumb: "If the process is totally blocked by a particular inhibitor, then the protein/gene blocked by the inhibitor would be annotated to the process.  If the process was partially blocked by the inhibitor, then the protein/gene would be annotated to regulation of the process."

**Comment by pgaudet** [comment]
@vanaukenk 
Is this our current rule? I think we used to say that years ago but I think this is not the current rule of thumb. 

Thanks, Pascale

**Comment by vanaukenk** [comment]
@slaulederkind @pgaudet 
I honestly don't remember that rule of thumb for GO annotation, but worry that applying that rule could potentially oversimplify the interpretation of experimental results.
Is the new term requested meant to be for annotating Fis1?

**Comment by slaulederkind** [comment]
Yes, the new term is for Fis1.

**Comment by pgaudet** [comment]
Hi @slaulederkind 

I created the new term: 

+id: GO:0140208
+name: apoptotic process in response to mitochondrial fragmentation
+namespace: biological_process
+def: "Any apoptotic process that occurs as a result of mitochondrial fragmentation." [PMID:18940801]
+synonym: "apoptosis in response to mitochondrial fragmentation" EXACT []
+is_a: GO:0006915 ! apoptotic process

With respect to 'regulation': How we have decided to address this at the Cambridge meeting (this was first discussed in Texas a few years back) is to use new qualifiers that describe how direct is the role of a protein within the process. For this case you would use 'causally upstream of'.  Hopefully that's OK with you. The qualifiers should be available by the next GO meeting (May 2018). 

Thanks, Pascale
